include/CMakeLists.txt

```diff
@@ -6,13 +6,13 @@
 #
 add_library(simdjson-headers INTERFACE)
 target_compile_features(simdjson-headers INTERFACE cxx_std_11) # headers require at least C++11
-target_include_directories(simdjson-headers INTERFACE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>)
+target_include_directories(simdjson-headers INTERFACE
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+  $<INSTALL_INTERFACE:include/>)
```

Comment:
Will ${CMAKE_CURRENT_BINARY_DIR} work here instead?

Reply:
CMAKE_CURRENT_BINARY_DIR doesn't work since it needs the installation path. {CMAKE_INSTALL_INCDIR} might be more explicit and in line with GNUInstalldirs, though CMake Documentation states 
>  Relative paths are allowed within the INSTALL_INTERFACE expression and are interpreted relative to the installation prefix